### DISCUSSION TITLE
Reverse Chronological Order

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#4113] [Help]
Hello! Can I set the data list into fixed reverse chronological order?

**Comment by mjashanks** [comment]
On a Data Provider (or block component) use the `Sort Column` and `Sort Order` settings.

In my example below, `Sort Column: Followup Date` and `Sort Order: Descending` will list records with the most recent date first.

![image](https://user-images.githubusercontent.com/3524181/150311043-460d2f12-d5ed-463e-b79e-7a1f584832d7.png)
